Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -827,15 +831,23 @@ async def slack_loop():
         client = SocketModeClient(
             app_token=APP_TOKEN,
             web_client=ASYNC_CLIENT,
-            logger=slack_logger  # type: ignore
+            logger=slack_logger,  # type: ignore
+            auto_reconnect_enabled=False
         )
         client.aiohttp_client_session = session
         client.socket_mode_request_listeners.append(listen)  # type: ignore
         try:
             await client.connect()
             await asyncio.sleep(float("inf"))
         except Exception as e:
-            await handle_listen_error(f"An error occurred {str(e)}")
+            demisto.debug(f"Exception in long running loop, waiting {exception_await_seconds} - {e}")
+            await asyncio.sleep(exception_await_seconds)
+            exception_await_seconds *= 2
```

Comment:
if a functioning integration may get to this line on a regular bases, sy once a day it will disconnect(not sure if that is possible), than maybe you need to initialize `exception_await_seconds` after a long time.